x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -390,6 +415,21 @@ static RoleDescriptor kibanaSystem(String name) {
                         TransportUpdateSettingsAction.TYPE.name()
                     )
                     .build(),
+                RoleDescriptor.IndicesPrivileges.builder()
+                    .indices("logs-wiz.vulnerability-*")
+                    .privileges("read", "view_index_metadata")
+                    .build(),
+                RoleDescriptor.IndicesPrivileges.builder()
+                    .indices("security_solution-wiz.vulnerability_latest-*")
+                    .privileges(
+                        "create_index",
+                        "read",
+                        "index",
+                        "delete",
+                        TransportIndicesAliasesAction.NAME,
+                        TransportUpdateSettingsAction.TYPE.name()
+                    )
```

Comment:
For indices not owned by Kibana, the `kibana-system` user should only have `read` privileges. `security_solution-*` falls under that pattern.

Can you please provide any additional information on why the elevated privileges are needed? I can then raise it with the KB security team.

Reply:
The Transform we're adding in the [corresponding PR](https://github.com/elastic/integrations/pull/10895) reads from the `logs-wiz.vulnerability-default` index and writes the result to the `security_solution-wiz.vulnerability_latest-v1` index. This index is created in Kibana after the integration installation, which is why these privileges are necessary.

We've implemented the same logic for our native integration for the same reason—creating an index via the Transform, as shown [here](https://github.com/elastic/elasticsearch/blob/91be7257fe2ea7e370d4060e098390e83f5cccf5/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java#L403).